airflow/providers/papermill/operators/papermill.py

```diff
@@ -57,36 +59,43 @@ class PapermillOperator(BaseOperator):
     def __init__(
         self,
         *,
-        input_nb: str | None = None,
-        output_nb: str | None = None,
+        input_nb: str | NoteBook | None = None,
+        output_nb: str | NoteBook | None = None,
```

Comment:
may be we should also make this fields mandatory

Reply:
I think it's OK to check their existence in the constructor.

Reply:
Previously it checked in execute, no idea why, just assume that is somehow related to Linage